src/document.rs

```diff
@@ -14,15 +14,14 @@ use pyo3::{
 
 use chrono::{offset::TimeZone, NaiveDateTime, Utc};
 
-use tantivy::{self as tv, schema::Value};
+use tantivy::{self as tv, schema::document::OwnedValue as Value};
```

Comment:
I don't like the alias to `Value` here.  If we decide to proceed with the approach in this PR, I'll change all the references to `OwnedValue` throughout the code.

Reply:
(These aliases in this PR are simply to make the changeset small, to get feedback on the approach)

Reply:
I'm not going to change these alias in this PR. If we want to change it, it can come in a future PR.